pytorch/jobs/docker/2.2/py3/requirements.txt

```diff
@@ -14,11 +14,11 @@ decorator==5.2.1
 h5py==3.14.0
 ipykernel==6.30.1
 jinja2==3.1.6
-keras==2.15.0  # keras uses the v2 api
+keras==3.11.3
 markupsafe==3.0.2
 matplotlib==3.10.1
-numpy==1.26.4
-openfermion==1.7.0
+numpy==1.26.4  # pin for pytorch 2.2 compatibility
```

Comment:
Is a PyTorch upgrade going to be another PR (since the Braket SDK already supports numpy 2.x)?

Reply:
We should look to modernize here. Also, examples should get numpy bumped - the `cirq` blocker for numpy is no longer relevant

Reply:
Yes, but I don't see a base image for a newer PyTorch version that is on Python 3.10: https://github.com/aws/deep-learning-containers/blob/master/available_images.md.

So we may have to do this along with upgrading the Python version.

Reply:
Never mind, the image exists, it's just not listed on that page. I opened another draft PR to test updating the PyTorch version: https://github.com/amazon-braket/amazon-braket-containers/pull/396